client/my-sites/plans-features-main/hooks/use-experiment-for-trail-map.ts

```diff
@@ -1,45 +1,47 @@
 import config from '@automattic/calypso-config';
+import {
+	isTrailMapAnyVariant,
+	isTrailMapCopyVariant,
+	isTrailMapStructureVariant,
+	setTrailMapExperiment,
+	TrailMapVariantType as VariantType,
+} from '@automattic/calypso-products';
+import { useEffect } from 'react';
 import { useExperiment } from 'calypso/lib/explat';
-import type { DataResponse } from '@automattic/plans-grid-next';
 
-interface Props {
+interface Params {
```

Comment:
I tend to prefer the term `Params` for hooks, to differentiate from component `Props`.